### PR TITLE
Added a requirements.txt file and changed the installation process a bit

### PR DESCRIPTION
**Pull request by CAP1Sup**
Also added a .gitignore for dealing with .vscode folders. The new installation method should be faster and more streamlined.

### CODE REVIEW

**Comment by CAP1Sup** [comment]
I should have also added ```**/__pycache__``` to the .gitignore. That'll prevent any of the Python caches from being copied when you commit.

**Comment by wyatt** [comment]
Was literally about to pull request the same thing lol

File: README.md [hunk 1]
```diff
@@ -15,21 +15,14 @@ Firstly, we need to make sure you have python3 and pip3 already installed on you
...
-python3 -m pip install discord
-```
-
 4) Next, download the zip file for the Among Us Discord Bot <a href="https://bit.ly/3mbCzTx" download>here</a> and unzip it
 5) Navigate to the Among Us folder in the command prompt using cd (e.g. cd Downloads\AmongUsBot-master\)
-6) Run "python3 test.py" and see if it runs without any errors. You can test by fullscreening the EXAMPLE IMAGE (not the test.py window) and see the screen ratios. Make sure the "IMPOSTER" fits nicely inside the window. If by chance it doesn't, you can adjust settings in the config.py in the modules folder. Press "q" to quit the window.
-5) Okay finally, watch this tutorial to finish this one time setup! 
+6) Copy and paste the text below into your command prompt and wait for them all to install
+``` 
+python3 -m pip install -r requirements.txt
```
Comment (wyatt) [inline README.md:22]: I think you can just type `pip install -r requirements.txt`

**Comment by alpharaoh** [comment]
Thanks, I was going to do this later, but seems like you guys have kindly done it for me!

@CAP1Sup 
@wyatt

**Review by alpharaoh — COMMENTED** [review]
Thanks!

**Comment by CAP1Sup** [comment]
@wyatt
Yeah, you can use pip3 install -r requirements.txt. The problem is that sometimes the python3 installation and the pip3 installation can be different, so I like to use the python3 method to get rid of the possibility

**Pull request merged**